Infrastructure/Storage/FilesStorageHelper.cs

```diff
@@ -8,7 +8,9 @@
 namespace Infrastructure
 {
     public static class FilesStorageHelper
-    {        
+    {
+        public static readonly string DataDirectoryPath = $"{Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData)}\\TruckTir";        
```

Comment:
Our app name is PartsApp not TruckTir.
TruckTir is just a client's name (in our case the only client but it doesn't matter)

Reply:
edited